examples/lighting-app/nrfconnect/rpc.overlay

```diff
@@ -43,3 +43,6 @@ CONFIG_LOG_MODE_MINIMAL=n
 CONFIG_LOG_MODE_IMMEDIATE=y
 CONFIG_LOG_BACKEND_UART=n
 CONFIG_LOG_BACKEND_RTT=n
+
+# Increase zephyr tty rx buffer
+CONFIG_CONSOLE_GETCHAR_BUFSIZE=128
```

Comment:
The attribute service is not added to the nRF Connect apps although the config is changed. Is that deliberate?

Reply:
This is to fix a bug which has existed for a while related to occasional corruption on the tty bus for RPCs. It is caused by zephyr putting a '~' on the tx buffer when the RX buffer overflows. This is done in the low-level tty driver which messes up the HDLC encoding.

That being said, your point is valid that the attribute service could be added for nrf also, I made that change and tested it.

Reply:
Thanks!